tools/lint

```diff
@@ -145,6 +144,15 @@ def run() -> None:
         fix_arg=["--write"],
         description="Formats CSS, JavaScript, YAML",
     )
+    linter_config.external_linter(
+        "black",
+        ["black"],
+        ["py"],
+        description="Reformats Python code",
+        check_arg=["--check"],
```

Comment:
I assume this is the opposite of `--fix-arg`?  Don't remember it but sgtm.

Reply:
It is: zulip/zulint@e1817f0ac2fbf3babccce656956d999cc53c0cd5.